app/services/releasecop_service.rb

```diff
@@ -5,7 +5,7 @@ class ReleasecopService
 
   # format criteria: %h %ad %s (%an, %ae)
   # see https://github.com/joeyAghion/releasecop/blob/master/lib/releasecop/comparison.rb#L5
-  LOG_LINE_EXPR = /^(?<sha>[0-9a-f]+) (?<date>[0-9\-]+) (?<message>.*) \((?<name>.*), (?<email>.*)\)\w*$/.freeze
+  LOG_LINE_EXPR = /^(?<sha>[0-9a-f]+) (?<date>[0-9-]+) (?<message>.*) \((?<name>.*), (?<email>.*)\)\w*$/.freeze
```

Comment:
I'm not very good with Regex 😅 , was this a Rubocop suggestion?

Reply:
Yes this was a rubocop auto-correction. My usual upgrade routine contains ubdating the rubocop gem, including all new cops and then running `rubocop -A` to make sure to have the new Ruby syntax suggestions all in there.

Reply:
I usually go and try regexes out on https://rubular.com/